docs/UserGuide.md

```diff
@@ -929,11 +929,16 @@ _Details coming soon ..._
 
 ## Common Issues
 
+1. I updated CCA Manager and now nothing shows up in my contact list.
+   Solution: delete your `data` folder and re-launch CCA Manager. 
```

Comment:
Should we add this? Since this is asking the user to delete all of their data.

Reply:
Lets just put a warning then

Reply:
Yea that works